src/client/dfuse/dfuse.h

```diff
@@ -262,6 +262,7 @@ struct fuse_lowlevel_ops *dfuse_get_fuse_ops();
  * so set LARGEFILE here for debugging
  */
 #define LARGEFILE 0100000
+#define FMODE_EXEC 0x20
```

Comment:
Curious, what is FMODE_EXEC?

Reply:
It's a kernel flag, that maps onto the non-existent O_EXEC, we only see it in userspace because of fuse.  It means the open request is because the kernel is launching a binary.